lib/solid/parser.ex

```diff
@@ -137,6 +145,16 @@ defmodule Solid.Parser do
 
   end_comment = string("endcomment")
 
+  custom_tag =
+    ignore(opening_tag)
+    |> ignore(space)
+    |> lookahead_not(inbuilt_tags)
+    |> concat(filter_name)
```

Comment:
I was wondering... what if we had to receive the allowed custom tags that we want to be able to parse instead of parsing anything? 🤔 

This way we have a very clear set of tags we must parse?

What do you think?

Reply:
Oh, that's what I wanted to do! I'm very new to nimble_parsec so I had no idea how to do that, haha.